structuredefinitions/UKCore-HealthcareService.xml

```diff
@@ -4,9 +4,9 @@
   <url value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-HealthcareService" />
   <version value="1.3.0" />
   <name value="UKCoreHealthcareService" />
-  <title value="UK Core HealthcareService" />
+  <title value="UK Core Healthcare Service" />
```

Comment:
Don't think this should be changed because 'HealthcareService' is named as such in base standard.

Reply:
HL7 does not have a `title` value, they use `name` which has no whitespace like ours. 

Our Design & development approach states "title: UK Core _[Name]_ – _[Name]_ is in Proper Case", this was clarified and agreed in a recent DA meeting.

Hl7 also states: "A short, descriptive, user-friendly title for the structure definition." which agrees with our approach

Reply:
Regarding the version not changing i was following the HL7 versioning policy, which to me suits metadata changes better (and something me and Adam had been doing unofficially):

"When an artifact versioning policy is metadata:
- Non-substantive changes to the metadata elements of the artifact MAY be made without incrementing the version number, but
- When this occurs, the date element SHALL be updated
"